### PR TITLE
refactor(experimental): add tests for decompiling durable nonce transaction

### PR DESCRIPTION
**Pull request by mcintyre94**
This PR adds unit tests for decompiling a durable nonce transaction

### CODE REVIEW

**Comment by mcintyre94** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1776** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1776" target="_blank">Graphite</a> 
    * **PR #1777** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1777" target="_blank">Graphite</a> 
      * **PR #1779** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1779" target="_blank">Graphite</a> 
        * **PR #1780** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1780" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1780?utm_source=stack-comment).

File: packages/transactions/src/__tests__/decompile-transaction-test.ts [hunk 1]
```diff
@@ -316,4 +317,357 @@ describe('decompileTransaction', () => {
...
+                            accountIndices: [
+                                1, // nonce account address
+                                3, // recent blockhashes sysvar
+                                0, // nonce authority address
+                            ],
+                            data: new Uint8Array([4, 0, 0, 0]),
+                            programAddressIndex: 2,
+                        },
+                        { programAddressIndex: 4 },
+                        { programAddressIndex: 5 },
```
Comment (buffalojoec) [inline packages/transactions/src/__tests__/decompile-transaction-test.ts:472]: Prob not a big deal, but should be include `accountIndices` here for extra testing?

**Review by buffalojoec — APPROVED** [review]
Lgtm!

File: packages/transactions/src/__tests__/decompile-transaction-test.ts (hunk 1 above)
Comment (mcintyre94) [reply packages/transactions/src/__tests__/decompile-transaction-test.ts]: Yep good shout, added some accounts/data to one of the extra instructions. It should be covered by the same code as the conversion of the durable nonce instruction, but that's a bit of an implementation detail so worth adding to another instruction.

**Pull request merged**